scripts/roles.json

```diff
@@ -3971,30 +4364,29 @@
     "prohibitedProps": [],
     "props": [
       "aria-atomic",
+      "aria-braillelabel",
+      "aria-brailleroledescription",
       "aria-busy",
       "aria-checked",
       "aria-controls",
       "aria-current",
       "aria-describedby",
+      "aria-description",
       "aria-details",
       "aria-disabled",
       "aria-dropeffect",
-      "aria-errormessage",
       "aria-expanded",
       "aria-flowto",
       "aria-grabbed",
       "aria-haspopup",
       "aria-hidden",
-      "aria-invalid",
       "aria-keyshortcuts",
       "aria-label",
       "aria-labelledby",
       "aria-live",
       "aria-owns",
       "aria-posinset",
-      "aria-readonly",
```

Comment:
Still listed and the next removed prop as well: https://www.w3.org/TR/wai-aria-1.2/#menuitemradio